docker-compose.yml

```diff
@@ -50,6 +50,8 @@ services:
       - '8123:8123'
       - '8443:8443'
       - '9000:9000'
+    volumes:
+      - ch-data:/var/lib/clickhouse/
```

Comment:
Was that addition because of [this comment](https://github.com/deeppavlov/dialog_flow_framework/pull/141#discussion_r1302987506)?

I meant to add a volume for `dashboard` so that it can be removed for new dashboard config to be uploaded.

Also, why were other volumes removed in 211b8db478df2ee22ec40704b9e996dd357729cc? I think that having no anonymous volumes is a good thing.

Reply:
Right now, we have no volumes in docker-compose.yml in dev. I don't know for what reason. But it would probably make sense to add them back in some other pull request, if we need them.

Reply:
We can also drop the existing assets in superset through the API, which is probably a nicer solution.

Reply:
> We can also drop the existing assets in superset through the API, which is probably a nicer solution.

I just noticed that dashboard doesn't create anonymous volumes and whenever I restart dashboard container it no longer holds any config. So the solution may not be necessary.

The new function `drop_superset_assets` doesn't work for me (not for changing chart names nor for changing database connection config).
Could you either remove it or debug it and add tests?

Reply:
Updated